src/main/java/edu/hm/webtec/entities/Topic.java

```diff
@@ -0,0 +1,43 @@
+package edu.hm.webtec.entities;
+
+import org.hibernate.validator.constraints.NotEmpty;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+
+
+/**
+ * A topic describes what subjects a question covers. A question can have multiple topics, for example "java" and "math".
```

Comment:
Du hast wahrscheinlich "Exercise" gemeint mit "question" oder? Wenn du das dann noch in der Form schreibst:
{@link Exercise}, dann hast du im Javadoc auch gleich den Link zur Exercise (kannst du auch schon machen, wenn es die Entität noch nicht gibt, wirft keinen Fehler). Am besten immer alle Klassen, die man im Javadoc verwendet verlinken.